### PR TITLE
LPS-159752 Add a Preview to the Info Panel of Web Content in the Content Dashboard	

### PR DESCRIPTION
**Pull request by cgoncas**
### Motivation

As a content marketer, I want to view a preview of a web content in the info panel, so that I can more thoroughly understand the context of the asset at a glance.

[LPS-159752](https://issues.liferay.com/browse/LPS-159752
)
### Proposed Solution

Add previewImage to the JournalArticleInfoItemFieldValuesProvider.
Read that value in a new PreviewJournalArticleContentDashboardAction of PREVIEW_IMAGE action.
Implement Preview method inside JournalArticletContentDashboardItem.

### How to test it

1. Create a web content with a Small Image.
2. Go to the Content Dashboard and open de info panel.
3. The image should be displayed in the info panel.

### Screencaps

Captura de pantalla 2022-08-24 a las 12 45 32

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:tango

**Comment by cgoncas** [comment]
ci:reevaluate:1287401_4073

**Comment by cgoncas** [comment]
ci:reevaluate:1287401_4073

**Comment by cgoncas** [comment]
ci:reevaluate:1287401_4073

**Comment by cgoncas** [comment]
ci:reevaluate:1287401_4073

**Comment by cgoncas** [comment]
ci:reevaluate:1287401_4073

**Comment by cgoncas** [comment]
ci:reevaluate:1287401_4073

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/info/item/provider/JournalArticleInfoItemFieldValuesProvider.java [hunk 1]
```diff
@@ -239,6 +239,11 @@ private List<InfoFieldValue<Object>> _getJournalArticleInfoFieldValues(
 					themeDisplay);
 
 				if (Validator.isNotNull(articleImageURL)) {
+					journalArticleFieldValues.add(
```
Comment (jkappler) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/info/item/provider/JournalArticleInfoItemFieldValuesProvider.java:242]: @cgoncas, can't we use the existing field smallImageInfoField for now and not create a new one? I know that we want to have standardized fields in the future, and we would need to adapt it later on. What do you think?

**Comment by jkappler** [comment]
@cgoncas, other than the question all looks good to me :)

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/info/item/provider/JournalArticleInfoItemFieldValuesProvider.java (hunk 1 above)
Comment (cgoncas) [reply modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/info/item/provider/JournalArticleInfoItemFieldValuesProvider.java:242]: Hey @jkappler , we have added this new field to blogs and documents and media, only standardized names are being used in the contend dashboard, for now.

We did the same with publish date, instead display date.

**Review by jkappler — APPROVED** [review]

**Comment by jkappler** [comment]
Perfect @cgoncas!

**Comment by jkappler** [comment]
ci:test:relevant

**Pull request closed without merging**